BetterSpecs/SpecObject.cs

```diff
@@ -22,19 +22,25 @@ public abstract class SpecObject
 
         internal virtual void Invoke(Action action, string text)
         {
-            text = text.PadLeft(_ident + text.Length, ' ');
-
             Console.OutputEncoding = System.Text.Encoding.UTF8;
-            Console.WriteLine(text);
+            Console.WriteLine(Indent(_ident) + text);
             action.Invoke();
         }
+        private string Indent(int count)
+        {
+            return "".PadLeft(count);
+        }
     }
 
     public class Let
     {
         private readonly Dictionary<string, object> _values = new Dictionary<string, object>();
+
+        //TODO: This variable will be removed coming soon.
```

Comment:
Probably, we can't remove this Dictionary. There are cases where performance is a importante attribute! Well, the instance of object only exists when invoke itself!